docs/data/system/experimental-api/extend-sx-prop/extend-sx-prop.md

```diff
@@ -0,0 +1,24 @@
+# Extend the `sx` prop
+
+<p class="description">Learn about the experimental API for extending or changing the behavior of the `sx` prop.</p>
+
+## Extend the `sx` prop
+
+If you wish to add new keys that would be processed by the `sx` prop, you can do that by extending the `unstable_sxConfig` option inside the theme.
+
+{{"demo": "ExtendTheSxProp.js" }}
+
+## Override some of the existing behavior
+
+There is also a possibility to change some of the existing behavior. You can simply provide a custom config in the for the specific key.
+
+{{"demo": "ChangeTheBehaviorSxProp.js" }}
```

Comment:
What exactly is this demo/section describing that's different from the previous section? Is it that you can create your own custom values for the sx CSS keys? I think it could be a bit more clear.

Reply:
It's about changing how certain CSS keys are processed. For example by default the `borderRadius` value is being multiple by the default theme value, but the demo shows how you can add more keys and use them directly. I will try to rephrase.

Reply:
I've changed the writing, I hope it is clearer now :)